.gitignore

```diff
@@ -0,0 +1,6 @@
+.vscode
+target
+src/test/java/org/apache/sling/commons/json/CVE.java
```

Comment:
Please use a standard .gitignore file from one of our repos, e.g. https://github.com/apache/sling-org-apache-sling-api/blob/master/.gitignore . This makes it compatible with multiple IDEs.